lib/service/service_test.go

```diff
@@ -427,6 +427,32 @@ func TestGetAdditionalPrincipals(t *testing.T) {
 	}
 }
 
+// TestDesktopAccessFIPS makes sure that Desktop Access can not be started in
+// FIPS mode. Remove this test once Rust code has been updated to use
+// BoringCrypto instead of OpenSSL.
+func TestDesktopAccessFIPS(t *testing.T) {
+	t.Parallel()
+	fakeClock := clockwork.NewFakeClock()
+
+	// Create and configure a default Teleport configuration.
+	cfg := MakeDefaultConfig()
+	cfg.AuthServers = []utils.NetAddr{{AddrNetwork: "tcp", Addr: "127.0.0.1:0"}}
+	cfg.Clock = fakeClock
+	var err error
+	cfg.DataDir, err = ioutil.TempDir("", "teleport")
```

Comment:
```suggestion
	cfg.DataDir = t.TempDir("", "teleport")
```

Then no need for an error check or to defer the removal.